plugins/Kaleidoscope-IdleLEDs/src/kaleidoscope/plugin/IdleLEDs.cpp

```diff
@@ -44,26 +44,38 @@ void IdleLEDs::setIdleTimeoutSeconds(uint32_t new_limit) {
   idle_time_limit = new_limit * 1000;
 }
 
+void IdleLEDs::resume() {
+  // Enabling LEDs is fairly expensive, so we only do it if we have to.
```

Comment:
In what way is it expensive to enable them? I guess running an animated effect is expensive, but there's a similar comment around disabling, so I'm not sure what this comment is referring to.

Reply:
Ah. It might make sense to push that logic down into `LEDControl`, then. Maybe that should be a separate PR.